client/verta/verta/endpoint/build/_build.py

```diff
@@ -102,3 +102,30 @@ def get_scan(self) -> _build_scan.BuildScan:
 
         """
         return _build_scan.BuildScan._get(self._conn, self.id)
+
+    def start_scan(self, external: bool = False) -> _build_scan.BuildScan:
+        """Start a new scan for this build. Internal scans are not yet supported.
+        Use ``external=True`` parameter.
+
+        This function only awaits a success response for the request, not any result
+        from the scan itself. Use
```

Comment:
It's a bit vague what "the request" is or does since it's an implementation detail of our code (and we don't technically wait for a _success_, either). Maybe something like

```suggestion
        This function only starts a scan; it does not wait for the finished scan to
        return results. Use
```

Reply:
Done.